src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/timeline/service/TimelinePersistenceService.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.service
+
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.Timeline
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.TimelineEvent
+
```

Comment:
"PersistenceService" is the terminology I introduced in OneTimeCode; though I did think of a better name for it the other day (and I've now forgotten it !!!)

Reply:
I've remembered it now .... how about `PersistenceAdapter` ? Because thats actually what it does (adapts the persistence concern from the app - which could be jpa, or mongo, or flat file, etc - into the domain)
"Adapter" also aligns with some of the terminology from the Hexagonal Architecture that we are basing this on (ie. "ports and adapters")

So ... shall we use the term "persistence adapter", in which case this interface becomes `TimelinePersistenceAdapter` ?

What do you think?

Reply:
`PersistenceBridge` 😉 

Just kidding, LGTM.